xbmc/cores/dvdplayer/DVDCodecs/Overlay/DVDOverlayCodecFFmpeg.cpp

```diff
@@ -237,10 +239,28 @@ CDVDOverlay* CDVDOverlayCodecFFmpeg::GetOverlay()
     if(m_Subtitle.rects[m_SubtitleIndex] == NULL)
       return NULL;
 
-    AVSubtitleRect& rect = *m_Subtitle.rects[m_SubtitleIndex];
+    AVSubtitleRect rect = *m_Subtitle.rects[m_SubtitleIndex];
     if (rect.pict.data[0] == NULL)
       return NULL;
 
+    m_height = m_pCodecContext->height;
+    m_width  = m_pCodecContext->width;
+
+    RENDER_STEREO_MODE render_stereo_mode = g_graphicsContext.GetStereoMode();
+    if (render_stereo_mode != RENDER_STEREO_MODE_OFF)
```

Comment:
Root problem is that if bitmap subtitles are "3D encoded" (i.e. already doubled), you get it 4 times.
In that case, the subtitle bitmap has, in the SBS case, a width > vid width / 2. 

That's obviously not foolproof, as a 2D subtitle will also have a width > vid width / 2. Works ok for TAB and height, though.
I have to think of something...